dalesbred/src/main/java/org/dalesbred/transaction/DefaultTransaction.java

```diff
@@ -59,7 +59,7 @@
 
             } catch (Exception e) {
                 connection.rollback();
-                log.log(Level.WARNING, "rolled back transaction because of exception: " + e, e);
+                log.warn("rolled back transaction because of exception.", e);
```

Comment:
There's a slight change in semantics here: previously the message's toString was part of the log message proper. One could of course argue that this is undesirable, but I think it would be better to implement a straight 1-1 port without any changes to semantics here.